content/diversity_and_inclusion.md

```diff
@@ -0,0 +1,9 @@
+# What it means
+
+# Why it is important
+
+# Understanding why it is a difficult problem
+
+# Our framework
+
+## The four pillars
```

Comment:
Is this section where we're going to describe what each of the pillars are with their goals, do you want that to be under each pillar respectively, or both?

Reply:
I was thinking this section about how the four pillars flow, dependent on each other, from a high level. As we potentially add more pillars having a section that explains how they interact becomes more important, I think. Will add a sub-description explaining that if you agree. Otherwise we can remove and have them self contained.

Reply:
Agree to the sub-description. Let me know if you need any help on drafting explanations.